src/System.Security.Cryptography.X509Certificates/tests/ChainTests.cs

```diff
@@ -264,5 +264,39 @@ public static void SafeX509ChainHandle_InvalidHandle_StaticObject()
 
             Assert.Same(firstCall, secondCall);
         }
+
+        [Fact]
+        [OuterLoop( /* May require using the network, to download CRLs and intermediates */)]
```

Comment:
Added an Offline check for immediately after the Online check.  (In the same method so that ordering is guaranteed).  The Offline data is validated to be the same as the Online data.